tests/foreman/ui/test_partitiontable.py

```diff
@@ -122,8 +122,6 @@ def test_remove_partition_table(self, name):
                                 os_family=os_family)
             self.assertIsNotNone(self.partitiontable.search(name))
             self.partitiontable.delete(name, really=True)
-            self.assertIsNotNone(self.partitiontable.wait_until_element
```

Comment:
Success notification appears on UI, but it disappears so quickly and selenium is unable to catch it. Anyway we are asserting the deleted entity via search.